hazelcast/src/main/java/com/hazelcast/config/DeviceConfig.java

```diff
@@ -27,4 +29,9 @@
      * otherwise, returns {@code false} if this configuration is for a remote device.
      */
     boolean isLocal();
+
+    /**
+     * Returns the device capacity.
+     */
+    MemorySize getCapacity();
```

Comment:
I'd move the methods and fields from `MemorySize` to a new `Capacity` class and would make `MemorySize` extending `Capacity`, as well as marking it `@Deprecated`. We now use it not only for the native memory size, but for the hybrid log's memory and device tiers' capacity as well. Potentially in a follow-up PR. WDYT? cc @vbekiaris

Reply:
@blazember good suggestion, +1

Reply:
Thanks for the suggestion, addressed in https://github.com/hazelcast/hazelcast/pull/20523/commits/d80acd7191b204815139ceed66af50f14a0539f0